test/functional/page_objects/settings_page.js

```diff
@@ -666,6 +666,10 @@ export function SettingsPageProvider({ getService, getPageObjects }) {
       await checkBox.click();
       return await this.canSavedObjectsBeDeleted();
     }
+    async isOss() {
```

Comment:
@kertal I like this method and although it works for on-prem, it won't work for cloud since there is no license management.